lib/jsxgettext.js

```diff
@@ -53,6 +53,9 @@ function getTranslatable(node, options) {
   if (options.keyword.indexOf(funcName) === -1)
     return false;
 
+  if (arg && funcName.substr(0, 1) === "n" && (isStrConcatExpr(arg) || isStringLiteral(arg)) && node.arguments[1] && (isStrConcatExpr(node.arguments[1]) || isStringLiteral(node.arguments[1])))
```

Comment:
Heh, checking the first letter of the function name is a good start but may not be very good in the future. Anyways, let's add a comment above stating what this does for potential newcomers ;)